File: LorePdks.COMMON/DTO/Kisi/KisiDTO.cs
using LorePdks.COMMON.DTO.Base;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LorePdks.COMMON.DTO.Common
{


    public class KisiDTO : BaseDTO
    {

        public string ad { get; set; }
        public string soyad { get; set; }
        public string tc { get; set; }
        public string cepTel { get; set; }
        public string email { get; set; }
        public string loginName { get; set; }
        public string sifre { get; set; }

    }
}

File: LorePdks.WEB/src/app/core/models/KisiDto.ts
import { baseDto } from "./baseDto";

/**
 * Kullanıcı bilgisini temsil eden model
 */
export class KisiDto extends baseDto {
  ad: string;
  soyad: string;
  tc?: string;
  cepTel?: string;
  email?: string;
  loginName: string;
  sifre?: string;
}